src/styles/nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+  display: flex;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__list-el {
+    display: flex;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__list-link {
+    line-height: 60px;
+    text-decoration: none;
+    color: var(--main-letters-color);
+    position: relative;
+    font-weight: 500;
+
+    &:hover {
+      color: var(--button-color);
+    }
+  }
+}
+  
+.is-active {
+  color: var(--button-color);
+
+  &::after {
+    content: "";
+    position: absolute;
+    width: 37px;
```

Comment:
Do not hardcode `width` here, it will not work on other items

<img width="670" alt="Screenshot 2023-05-14 at 12 15 10" src="https://github.com/mate-academy/layout_catalog/assets/32853034/5d2ed476-fed9-4a29-a121-13329131ebe3">